.gitignore

```diff
@@ -21,3 +21,5 @@ __pycache__/
 tests/test-report.xml
 tests/functional-test-report.xml
 .orig
+
+.python-version
```

Comment:
Should the python-version file be ignored from the repository?

Reply:
This file is created when you run the setup scripts and will enforce using the 'apigee' venv when you cd into the repo. But if for some reason you change it, you don't want to commit it by accident -- if someone pulls that commit they would start using whatever venv is there. That's why I ignored it.

Reply:
Ah I see, that makes sense yup 👍